collections/_infographics/emissions/emissions-eu-trends.md

```diff
@@ -13,4 +13,38 @@ data-orig:
+* The infographic shows <glossary id="anthropogenicgreenhousegases">anthropogenic CO<sub>2</sub> emissions</glossary> and emissions of N<sub>2</sub>O, CH<sub>4</sub>, HFC, PFC, SF<sub>6</sub> and NF<sub>3</sub> expressed in units of CO<sub>2</sub> equivalent (CO<sub>2</sub>eq). This unit takes into account the long-term effect of greenhouse gases in the atmosphere and converts them into an amount of CO<sub>2</sub> that would have the same warming effect. For more, see the article on [Global warming potential](https://en.wikipedia.org/wiki/Global_warming_potential).
+* The left part of the graph shows the absolute emissions of the European Union (these are emissions of the EU-27, i.e. countries that were EU members by 2021) in units of CO<sub>2</sub>eq over the last three decades. The different areas of human activity are distinguished by colour. On the right-hand side, the relative change in emissions compared to 1990 is indicated for each sector.
+* In 1990, emissions in what is now the European Union amounted to 5 billion tonnes of CO<sub>2</sub>eq. In 2019, it was 3.7 billion tonnes. This statistic indicates that EU emissions have fallen by 24% over nearly thirty years. Total EU emissions have been falling more significantly since 2007, after the global financial crisis, in all sectors except transport.
+{% comment %}
+{% include preview-box.html
+    title="Where does the Czech Republic stand in terms of emissions?"
+    text="We have created a similar [infografiku](/infografiky/emise-cr-vyvoj) for the Czech Republic."
+    slug="emise-cr-vyvoj"
+%}
+{% endcomment %}
+
+## What we see in individual sectors
+
+* __Energy:__ Emissions from the energy sector fell by 39% compared to 1990 to 972 million tonnes of CO<sub>2</sub>eq per year. Emissions from the energy sector began to decline significantly after 2007. In recent years we have seen a more rapid decline, which can be expected to continue in the future, given the EU's commitment to achieving <glossary id="co2eq">climate neutrality</glossary> by 2050.
+Emissions in the energy sector come primarily from electricity and heat production. These emissions can be reduced through energy savings and the development of renewable and low-carbon energy sources. They also come from leakages from fuel extraction, fuel production and treatment and oil processing.
+* __Transport:__ Transport emissions increased by 33% compared to 1990 to 968 million tonnes of CO<sub>2</sub>eq per year. The detailed graph on the right shows a temporary drop in emissions after 2007 due to the global financial crisis and the subsequent economic recession. Since 2013, emissions can be seen to have risen again. GHG emissions from transport are primarily from the combustion of fossil fuels in the engines of road transport vehicles. (In 2019, 81% of all emissions from the transport sector were from transport, with aviation accounting for 15%.)
```

Comment:
1) add comma:
from
Transport emissions increased by 33% compared to 1990 to 968 million tonnes of CO<sub>2</sub>eq per year. 
to
Transport emissions increased by 33% compared to 1990, to 968 million tonnes of CO<sub>2</sub>eq per year. 

2) shorten:
from
Since 2013, emissions can be seen to have risen again. 
to
Since 2013, emissions have risen again. 

3) IMPORTANT:
Change
In 2019, 81% of all emissions from the transport sector were from transport, with aviation accounting for 15%.
to
In 2019, 81% of all emissions from the transport sector were from road transport, with aviation accounting for 15%.

Reply:
Thanks, I covered:
1. Added comma
2. Shortened the sentence.
3. Updated to "road transport". (IMPORTANT)

Reply:
@tjarnikova as this comment one is marked with "IMPORTANT", I will leave it open for you to resolve.